### PR TITLE
Add python-django-readonly-field package

### PR DESCRIPTION
**Pull request by sjha4**

### CODE REVIEW

File: packages/pulpcore/python-django-readonly-field/python-django-readonly-field.spec [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+License:        MIT
+URL:            https://github.com/peopledoc/django-readonly-field
+Source0:        https://files.pythonhosted.org/packages/source/d/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python3-devel
+BuildRequires:  python3-Django >= 1.11
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-tox
```
Comment (evgeni) [inline packages/pulpcore/python-django-readonly-field/python-django-readonly-field.spec:17]:
```suggestion
BuildRequires:  python3-django >= 1.11
BuildRequires:  python3-setuptools
```

File: packages/pulpcore/python-django-readonly-field/python-django-readonly-field.spec [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+%description
+%{summary}
+
+%package -n     python3-%{pypi_name}
+Summary:        %{summary}
+%{?python_provide:%python_provide python3-%{pypi_name}}
+Requires:       python3-Django >= 1.11
```
Comment (evgeni) [inline packages/pulpcore/python-django-readonly-field/python-django-readonly-field.spec:25]:
```suggestion
Requires:       python3-django >= 1.11
```

**Comment by sjha4** [comment]
@evgeni : Does this look ok now?

**Comment by jlsherrill** [comment]
```
DEBUG util.py:598:  Error: No Package found for python3-tox
```
need another dep?

**Comment by sjha4** [comment]
That was not a pulp_rpm dependency when I installed locally..let me check and update the spec..

**Comment by jlsherrill** [comment]
LGTM  @evgeni what about you?

**Comment by jlsherrill** [comment]
I got it!

**Pull request merged**